File: 0060-permutation-sequence/0060-permutation-sequence.cpp
class Solution {
public:
    string getPermutation(int n, int k) {
        string s = "";
        for (int v = 1; v <= n; ++v) {
            s += to_string(v);
        }
        
        vector<string> permutationsList;
        
        do {
            permutationsList.push_back(s);
        } while (next_permutation(s.begin(), s.end()));
        
        return permutationsList[k - 1];
    
    }
};

File: 0060-permutation-sequence/0060-permutation-sequence.py

#from itertools import permutations
class Solution:
    def getPermutation(self, n: int, k: int) -> str:
        s = ""
        for v in range(1,n + 1):
            s += str(v)
            
        lst = ["".join(p) for p in permutations(s)] 

        return lst[k - 1]